lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -183,6 +183,8 @@ class LighthouseViewerReport {
       this._isNewReport = true;
 
       this.replaceReportHTML(JSON.parse(str));
+
+      ga('send', 'event', 'report', 'upload');
```

Comment:
There's a hidden file upload on the page as well as drag and drop. They both go through this callback.

 I originally used "view", but wanted to distinguish from the paste operation.